test/config.json

```diff
@@ -9,20 +9,6 @@
           "remote": false
         }
       ]
-    },
-    {
-      "file": "test/specs/multipleBrowsersDocumentUpload.js",
-      "timeout": 50000,
-      "browsers": [
-        {
-          "browserName": "IE",
-          "browser_version": "11.0",
-          "os": "Windows",
-          "os_version": "10",
-          "resolution": "1920x1080",
-          "remote": true
-        }
-      ]
```

Comment:
Didn't notice it in previous PR(s), but why did we remove the IE11 UI tests? 🤔

Reply:
I think that's a mistake. I'll enable them again